### PR TITLE
Migrate to tracker3.  JB#54233 OMP#JOLLA-140

### PR DESCRIPTION
**Pull request by pvuorela**
Recreated https://git.sailfishos.org/mer-core/qtdocgallery/merge_requests/11

cc @okodron

### CODE REVIEW

**Review by adenexter — APPROVED** [review]
LGTM, I know meta-data write and type query features aren't there, but they're also not used, and the edits in particular will only survive until something triggers a reset so saying that worked before is questionable anyway.

**Comment by pvuorela** [comment]
Added some minor .spec cleanups.

**Comment by pvuorela** [comment]
On a quick test, the type query seemed to even create a sane query, getting the number of related items. But overall I'm a bit puzzled how the whole thing should be even used :)

**Comment by adenexter** [comment]
> On a quick test, the type query seemed to even create a sane query, getting the number of related items. But overall I'm a bit puzzled how the whole thing should be even used :)

Summary statistics? The main page of the media player has the number of songs, artists, albums... Document gallery requirements were never really more specific than the name, but one platform may have provided a equivalent set of data.

File: rpm/qtdocgallery.spec [hunk 1]
```diff
@@ -37,7 +36,7 @@ touch .git
 %qmake5  \
     tracker_enabled=yes \
     MODULE_VERSION=5.0.0
-make %{?jobs:-j%jobs}
+%make_build
 
 %install
 rm -rf %{buildroot}
```
Comment (okodron) [inline rpm/qtdocgallery.spec]: Not necessary any more.

File: rpm/qtdocgallery.spec [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Name:       qt5-qtdocgallery
 Summary:    Qt document gallery optional module
-Version:    5.0.0
+Version:    5.2.0
 Release:    1
 Group:      System/Libraries
```
Comment (okodron) [inline rpm/qtdocgallery.spec]: I guess it also can be dropped.

Comment (pvuorela) [reply rpm/qtdocgallery.spec]: Adjusted this and above and changes also the url to this repo

**Review by okodron — APPROVED** [review]

**Pull request merged**